OSInAppBrowserLib/RouterAdapters/OSIABWebViewRouterAdapter.swift

```diff
@@ -36,12 +41,13 @@ public class OSIABWebViewRouterAdapter: NSObject, OSIABRouter {
         
         let viewModel = OSIABWebViewModel(
             url: url,
-            self.options.toConfigurationModel().toWebViewConfiguration(),
-            self.options.allowOverScroll,
-            self.options.customUserAgent,
-            self.options.allowsBackForwardNavigationGestures,
-            uiModel: self.options.toUIModel(),
-            callbackHandler: self.callbackHandler
+            customHeaders: customHeaders,
```

Comment:
Any specific reason why we stopped using `self` or is it just a matter of preference?